packages/application-generic/src/usecases/get-feature-flag/get-use-merged-digest-id.use-case.ts

```diff
@@ -12,7 +12,7 @@ export class GetUseMergedDigestId extends GetFeatureFlag {
       value,
       fallbackValue
     );
-    const key = FeatureFlagsKeysEnum.IS_USE_MERGED_DIGEST_ID;
+    const key = FeatureFlagsKeysEnum.IS_USE_MERGED_DIGEST_ID_ENABLED;
```

Comment:
This FF key on the enum was missing the `_ENABLED` suffix and broke the tests. Only the `enum` key has been updated, not the environment variable - so this will not affect customers.